networks.Rmd

```diff
@@ -0,0 +1,259 @@
+# Networks {#networks}
+
+```{r setup, include = FALSE}
+source("common.R")
+columns(1, 2 / 3)
+set.seed(1)
+```
+
+Like maps and spatial data, networks and graphs occupy a special part of the visualization landscape, but whereas spatial data mostly differ from regular plotting in their use of projections, networks bring their own data structure as well as their own visualization paradigms to the table. Because of these complications networks are not directly supported in ggplot2. Several efforts over the years have tried to add this missing piece and in this chapter we will see how to use ggraph for network visualization. Other packages that offers some of the same functionality includes geomnet, ggnetwork, and GGally for regular network plots, and ggtree and ggdendro for tree visualization specifically.
```

Comment:
Can you please add links to the package names?

Reply:
Which links should we choose? Github, CRAN, pkgdown sites if any? Considering recent misfortunes with linking to domains not in the hands of companies I'd be weary of doing the third, though that is usually the most useful link

Reply:
pkgdown is safe I think. I think the risk of domain highjacking is low for most sites.